corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -19,13 +19,14 @@
         attr: collapsible ? {
                 role: 'button',
                 'aria-expanded': showChildren() ? 'true' : 'false',
-                'aria-labelledby': captionId()
+                'aria-labelledby': captionId(),
+                tabindex: ko.utils.unwrapObservable($data.caption_markdown) || caption() ? '0' : '-1'
```

Comment:
'tabindex' and tabindex are both the same?

Reply:
Yup, 'tabindex' and tabindex both are valid key names, `aria-labelledby` is wrapped in single quotes as it has `-` symbol in between which makes it a invalid as a key name string.

Reply:
Cool. Good to keep uniformity then, i guess keeping everything in single quotes sounds a good approach to keep things the same.